cylc/rose/__init__.py

```diff
@@ -77,7 +77,19 @@
 to the Cylc scheduler:
 
 ``ROSE_ORIG_HOST``
-   The plugin provides the hostname of the computer where the plugin runs.
+   Cylc scripts will provide this as the name of the host the script is run
+   on, unless it is set in the ``[env]`` section of the configuration file.
```

Comment:
> unless it is set in the ``[env]`` section

What!

Reply:
I just went and checked bc @dpmatthews asked me to.